bin/bundle

```diff
@@ -1,3 +1,3 @@
-#!/usr/bin/env ruby2.1
```

Comment:
It was ruby2.1 because I was using the brightbox ruby-ng binaries, so that's what it gave me.

But I really don't know.